miner/worker.go

```diff
@@ -81,7 +83,8 @@ const (
 )
 
 var (
-	commitTxsTimer = metrics.NewRegisteredTimer("worker/committxs", nil)
+	commitTxsTimer   = metrics.NewRegisteredTimer("worker/committxs", nil)
```

Comment:
no use of this one `commitTxsTimer`

Reply:
yes , it is not added by this pr.
This is the metrics that bsc already has.